sonic_installer/main.py

```diff
@@ -604,8 +604,10 @@ def install(url, force, skip_platform_check=False, skip_migration=False, skip_pa
             migrate_sonic_packages(bootloader, binary_image_version)
 
     # Finally, sync filesystem
-    run_command("sync;sync;sync")
-    run_command("sleep 3")  # wait 3 seconds after sync
+    run_command(["sync"])
```

Comment:
>run_command [](http://example.com/codeflow?start=4&length=11)

The behavior changes. if first command fails, it will exit.

Reply:
I think second command will run regardless of whether or not the first one succeeds? https://stackoverflow.com/questions/25669540/what-is-the-difference-between-double-ampersand-and-semicolon-in-linux

Reply:
The new implementation will exit if the first `sync` fail. Considering this use case, it is a minor issue.